layouts/partials/fragments/hero.html

```diff
@@ -1,22 +1,22 @@
-{{ $data := ($.Scratch.Get "data") }}
-{{ "<!-- Hero -->" | safeHTML }}
-{{ $bg := $data.background | default "secondary"}}
+{{ $hero := (.Resources.GetByPrefix "hero") }}
```

Comment:
Let's simplify this to: `{{ $hero := . }}`

Reply:
Also not sure about the change from `$data` to `$hero`. I agree it makes it cleaner to see, which fragments code one is looking at, but makes code sharing harder. Happy to leave it at `$hero` just wanted to understand the reasoning better.

Reply:
Yeah, the same reason why I changed it. But it can remain as data for better code sharing.

Reply:
Ok let's leave it at $hero or in general $<fragment. Makes code sharing more conscious.